src/Utils.php

```diff
@@ -384,17 +384,17 @@ function wf_crm_get_contact_relationship_types($type_a, $type_b, $sub_type_a, $s
       $reciprocal = ($t['label_a_b'] != $t['label_b_a'] && $t['label_b_a'] || $t['type_a'] != $t['type_b']);
       if (($t['type_a'] == $type_a || !$t['type_a'])
         && ($t['type_b'] == $type_b || !$t['type_b'])
-        && (in_array($t['sub_type_a'], $sub_type_a) || !$t['sub_type_a'])
-        && (in_array($t['sub_type_b'], $sub_type_b) || !$t['sub_type_b'])
+        && (in_array(strtolower($t['sub_type_a']), $sub_type_a) || !strtolower($t['sub_type_a']))
+        && (in_array(strtolower($t['sub_type_b']), $sub_type_b) || !strtolower($t['sub_type_b']))
       ) {
         $ret[$t['id'] . ($reciprocal ? '_a' : '_r')] = $t['label_a_b'];
       }
       // Reciprocal form - only show if different from above
       if ($reciprocal
         && ($t['type_a'] == $type_b || !$t['type_a'])
         && ($t['type_b'] == $type_a || !$t['type_b'])
-        && (in_array($t['sub_type_a'], $sub_type_b) || !$t['sub_type_a'])
-        && (in_array($t['sub_type_b'], $sub_type_a) || !$t['sub_type_b'])
+        && (in_array(strtolower($t['sub_type_a']), $sub_type_b) || !strtolower($t['sub_type_a']))
+        && (in_array(strtolower($t['sub_type_b']), $sub_type_a) || !strtolower($t['sub_type_b']))
       ) {
         $ret[$t['id'] . '_b'] = $t['label_b_a'];
```

Comment:
This is fixing a bug very similar to what we noticed in the `ContactDedupeTest` -> originally the Student - Organization relationship did not show up as an option b/c `Student` did not match `student`.

Reply:
Do we always have `$sub_type_b` and `$sub_type_a` available in lower case? Given that in_array does a case sensitive comparison?

Reply:
A better solution would be to have an in_array non-case sensitive comparison of keys b/c we end up with `sub_type = Student` and `sub_type = student`, etc. We have seen that in a few places now. 

Implications are big: e.g. custom fields on Individual -> Student were not showing up (prior to using strtolower there); in this case you can not create a relationship between Individual -> Student and Organization b/c it does not think there are any relationships for Student (just for student).

I specifically created a sub_type -> Student (upper case) to test this. It's the most common scenario as well.

Reply:
Ok I've resolved this by ensuring we're always comparing contact sub_type in lower case 👆

Reply:
Will await test results before merging!

Reply:
Oh how did that fail? I need to take this back to my local.

Reply:
Ok I found a better place for the `strtolower()` 👍